src/js/components/sidebar.jsx

```diff
@@ -242,6 +283,16 @@ export default class SideBar extends React.Component {
     }
   }
 
+  _toggleMore() {
+    let moreActions = document.getElementById("moreActions");
```

Comment:
This is a React no-no. You never want to query for a component that has been mounted and modify it. This will interfere with React's reconciler. The way you keep track of mounted sub-components (i.e., elements) is through [refs](https://facebook.github.io/react/docs/refs-and-the-dom.html). However, keeping a ref and modifying its properties is not what you want to do.

Instead, you want to keep track of state of whether or not the more actions are expanded. To do this you need to do the following:

 1. Update the `state` property to add a new field, `moreActionsShown`, defaulting to `false`.
 2. Replace this function body with a call to `setState` with the new value.

```javascript
_toggleMore() {
    this.setState({ moreActionsShown: !this.state.moreActionsShown });
}
```

3. Update `render()` to conditionally add the `hidden` class to the `#moreActions` element when `state.moreActionsShown` is `false`.

Now, the call to `setState` in `_toggleMore` will trigger React to re-render and the new actions will be correctly shown (or hidden).

Rendering a React component with the same props and state should always have the same result.